packages/time-picker/src/TimePicker/components/MenuItem/MenuItem.module.scss

```
@@ -0,0 +1,33 @@
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/typography";
+
+.menuItem {
+  color: #2f2438;
+  background: transparent;
+  padding: 6px calc(#{$spacing-sm} - 2px);
```

```suggestion
  padding: $spacing-xs calc(#{$spacing-sm} - 2px);
```

Is there something else we could do instead the `-2px` on a spacing? Although I assume we'd probably be wanting a review from Jason at some point for design, so we can probably address it a little later once we're happy with functionality.

---

I have a feeling I might have copied these styles from somewhere. I'll have a look at where I found it from